Add more video examples, change examples layout

dayweek opened the pull request:
Added more videos based on suggestions 
I added examples to separate tabs. Feel free to change the layout back if not suitable

Commit:
Add more video examples, change examples layout

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

samhita-alla committed:
add additional examples

Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

samhita-alla commented:
@dayweek, thanks for creating the PR! Two videos were nearly ~40MB, so I deleted the files and added them as YouTube links. I hope that's fine with you. :)

dayweek commented:
No problem, thanks!

čt 13. 10. 2022 v 14:07 odesílatel Samhita Alla ***@***.***>
napsal:

> @dayweek <https://github.com/dayweek>, thanks for creating the PR! Two
> videos were nearly ~40MB, so I deleted the files and added them as YouTube
> links. I hope that's fine with you. :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/samhita-alla/geolocator/pull/7#issuecomment-1277435912>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABDHRLTVV2OYRZXGYVWGLDWC7UQPANCNFSM6AAAAAARDW2O2Y>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

samhita-alla committed:
change label

Signed-off-by: Samhita Alla <aallasamhita@gmail.com>
